plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -82,6 +99,35 @@ func calculateResourceOccupancy(pod *api.Pod, podRequests *schedulercache.Resour
 	}
 }
 
+// Calculate the resource used on a node.  'node' has information about the resources on the node.
+// 'pods' is a list of pods currently scheduled on the node.
+// TODO: Use Node() from nodeInfo instead of passing it.
+func calculateUsedPriority(pod *api.Pod, podRequests *schedulercache.Resource, node *api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
```

Comment:
Sure, 2 nodes with 16 cores. One is empty, the other one has 1 cpu requested.

Version A: 10-floor(10 * (16-0)/16) = 0 , 10-floor(10 * (16 - 1)/16) = 1
Version B: floor(10 * 0/16) = 0, floor(10 * 1/16) = 0

So the final ordering may be different. Sure we can reverse the least formula but the resulting formula would be unnecessary complicated.